### DISCUSSION TITLE
Adding a Filament form (select component) to a Livewire(v3.x) component

### DISCUSSION THREAD

**Discussion by danielveselinov** [livewire/livewire#8020] [1. Help] [answered]
Hi everyone,

I'm trying to implement Filament forms in my livewire component. Following the documentation I've done it successfully and it works fine, but as always there is a problem with my Select component.

What I'm trying to do?
I want to use filament select component with multiple choices connected with a relation, and it works fine, but when I refresh the page after saving the 'interests' in my case aren't present but they are synced in my DB correctly, and when I access the filament admin panel and preview the user there the same code works fine, the selected interested are filled (selected) in the input, but that's not the case in my Livewire component.

I'll pass the code if anyone can see what I'm doing wrong..

```php
class UpdateInterestForm extends Component implements HasForms 
{ 
  use InteractsWithForms;

  public User $user;
  public $interests;
  
  public function mount(): void
  {
      $this->user = Auth::user();
  }
  
  public function updateInterestUser(): void
  {
      $this->user->interests()->sync($this->interests);
  
      $this->dispatch('saved');
  }
  
  public function form(Form $form): Form
  {
      return $form
            ->schema([
                Select::make('interests')
                    ->multiple()
                    ->preload()
                    ->relationship('interests', 'name')
                    ->minItems(1)
                    ->maxItems(15)
            ])
            ->model(User::class);
  }
  
  public function render(): View
  {
      return view('livewire.profile.update-interest-form');
  }
}
```

Relation in the User model:

```php 
public function interests(): BelongsToMany { return $this->belongsToMany(Interest::class); }
```

Relation in the Interest model:

```php 
public function users(): BelongsToMany { return $this->belongsToMany(User::class); }
```

Note: this very code, Select::make('interests').. in my UserResource filament works just fine, the problem is in the Livewire component.

Filament - UserResource
![image](https://github.com/livewire/livewire/assets/87199149/b6546a27-209c-4a29-a970-08b18997ba49)

Livewire - 
![image](https://github.com/livewire/livewire/assets/87199149/2800e10d-f521-4659-91ba-1f0a36ef6656)

**Comment by joshhanley** [comment, marked as answer]
@danielveselinov I’d suggest asking this on the Filament repo or their discord server. 

And in future please don’t open a discussion and an issue. Just open a discussion. Thanks.
